text_generation/causal_lm/cpp/continuous_batching/library/CMakeLists.txt

```
@@ -41,9 +53,12 @@ add_library(openvino::continuous_batching ALIAS openvino_continuous_batching)
 target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src"
                                           PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
 target_compile_definitions(${TARGET_NAME} PRIVATE OPENVINO_TOKENIZERS_PATH="libopenvino_tokenizers.so")
-set_target_properties(${TARGET_NAME} PROPERTIES CXX_STANDARD 14 CXX_STANDARD_REQUIRED ON)
+set_target_properties(${TARGET_NAME} PROPERTIES CXX_STANDARD 17 CXX_STANDARD_REQUIRED ON)
+
+#need make_unique
+target_compile_features(${TARGET_NAME} PUBLIC cxx_std_17)
```

```suggestion
target_compile_features(${TARGET_NAME} PRIVATE cxx_std_17)
```
as `make_unique` is used internally.